api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -63,6 +64,14 @@ message HttpConnectionManager {
     envoy.api.v2.RouteConfiguration route_config = 4;
   }
 
+  oneof scoped_routes_specifier {
+    // Configuration for the Scoped RDS API which dynamically loads routing configuration scopes.
+    ScopedRds scoped_rds = 30;
```

Comment:
Would it be simpler if these options were just in the `route_specifier` oneof above? Why have them be orthogonal?

Reply:
My initial intention keeping them separate was to avoid confusion that SRDS and RDS are tightly coupled and require the same config source, etc.

However, based on the feedback in this review and reassessing this from the user's standpoint, I agree this is simpler if `ScopedRds` becomes part of the `route_specifier`, and `ScopedRds` expands to include both the `ConfigSource` for RDS and SRDS. This would allow me to remove `scoped_rds_template` from `Rds`, which avoids the confusion related to that field (I've never been quite happy with that field since I started working on this).